src/core_plugins/kibana/public/dashboard/directives/dashboard_panel.js

```diff
@@ -1,15 +1,19 @@
 import _ from 'lodash';
 import 'ui/visualize';
 import 'ui/doc_table';
-import { loadPanelProvider } from 'plugins/kibana/dashboard/components/panel/lib/load_panel';
+import 'plugins/kibana/dashboard/services/get_object_loaders_for_dashboard.factory';
 import FilterManagerProvider from 'ui/filter_manager';
 import uiModules from 'ui/modules';
 import panelTemplate from 'plugins/kibana/dashboard/components/panel/panel.html';
+import { getPersistedStateId } from 'plugins/kibana/dashboard/components/panel/lib/panel_state';
+import { loadSavedObject } from 'plugins/kibana/dashboard/components/panel/lib/load_saved_object';
 
 uiModules
 .get('app/dashboard')
-.directive('dashboardPanel', function (savedVisualizations, savedSearches, Notifier, Private, $injector) {
-  const loadPanel = Private(loadPanelProvider);
+.directive('dashboardPanel', function ($injector) {
+  const Private = $injector.get('Private');
```

Comment:
Well, `FilterManagerProvider` is still wrapped below. Isn't the only reason you get around using `Private` on `getObjectLoadersForDashboard` the fact that it has been globally registered under that name? That is still tightly coupled to Angular's DI, then.